src/javascript/app_2/App/Components/Elements/please_login_message.jsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes    from 'prop-types';
+import React        from 'react';
+import { localize } from '../../../../_common/localize';
+
+const PleaseLoginMessage = ({ onLogin, onSignup, children }) => (
```

Comment:
I don't think we need `please` in the name, `LoginMessage` is enough. same for the file name.

Reply:
i just though `LoginMessage` could be a bit ambiguous 😅

Reply:
why? it's a message for logging in

Reply:
TBH, this is the first time I've seen someone declare a constant in such polite fashion. ;)